apps/dashboard/app/models/files.rb

```diff
@@ -36,6 +36,13 @@ def self.stat(path)
     }
   end
 
+  # Returns if the File size exceeds Configuration.file_download_max
+  #
+  # @return [Boolean]
+  def self.downloadable?(path)
+    path.stat.size < Configuration.file_download_max
```

Comment:
See comment below. `file_download_max` should be an optional setting that could possibly be unset or `nil`. then this would become:

```
def self.downloadable?(path)
  Configuration.file_download_max ? (path.stat.size < Configuration.file_download_max) : true
end
```